app/controllers/Application.scala

```diff
@@ -598,7 +598,7 @@ object Application extends Controller {
   def appNameToDisplay(metadata: Option[Metadata], notebookPath: Option[String]): String = {
     val explicitName = metadata.map(_.name).getOrElse("Spark-notebook")
```

Comment:
previously this one was `Spark Notebook` instead of `Spark-notebook`. I like old one :)